CMakeLists.txt

```diff
@@ -36,6 +50,8 @@ if(NOT MSVC)
     endif()
 else()
     add_compile_options(/permissive-)
+    # We want to pass this flag to link.exe. In CMake 3.13, we can use add_link_options
+    # /IGNORE:4221 # LNK4221 warns for empty cpp files. We use such files.
```

Comment:
Did you try `link_libraries(-IGNORE:4221)`?

Reply:
Yes, `link_libraries` ignores the flag

Reply:
Passing the flag with `target_link_libraries` also ignores the flag, meaning the flag is not passed to `link.exe`